rgw/v2/tests/s3_swift/test_bucket_lc_object_exp_multipart.py

```diff
@@ -0,0 +1,235 @@
+            log.info("no of buckets to create: %s" % config.bucket_count)
+            # create bucket
+            for bc in range(config.bucket_count):
+                bucket_name = utils.gen_bucket_name_from_userid(
+                    each_user["user_id"], rand_no=1
+                )
+                bucket = reusable.create_bucket(bucket_name, rgw_conn, each_user)
+                life_cycle_rule = {"Rules": config.lifecycle_conf}
+                bucket_life_cycle = s3lib.resource_op(
+                    {
+                        "obj": rgw_conn,
+                        "resource": "BucketLifecycleConfiguration",
+                        "args": [bucket.name],
+                    }
+                )
+                put_bucket_life_cycle = s3lib.resource_op(
+                    {
+                        "obj": bucket_life_cycle,
+                        "resource": "put",
+                        "kwargs": dict(LifecycleConfiguration=life_cycle_rule),
+                    }
+                )
+                log.info("put bucket life cycle:\n%s" % put_bucket_life_cycle)
+                if put_bucket_life_cycle is False:
+                    raise TestExecError(
+                        "Resource execution failed: put bucket lifecycle failed"
+                    )
+                if put_bucket_life_cycle is not None:
+                    response = HttpResponseParser(put_bucket_life_cycle)
+                    if response.status_code == 200:
+                        log.info("bucket life cycle added")
+                    else:
+                        raise TestExecError("bucket lifecycle addition failed")
+                log.info("trying to retrieve bucket lifecycle config")
+                get_bucket_life_cycle_config = s3lib.resource_op(
+                    {
+                        "obj": rgw_conn2,
+                        "resource": "get_bucket_lifecycle_configuration",
+                        "kwargs": dict(Bucket=bucket.name),
+                    }
+                )
+                if get_bucket_life_cycle_config is False:
+                    raise TestExecError("bucket lifecycle config retrieval failed")
+                if get_bucket_life_cycle_config is not None:
+                    response = HttpResponseParser(get_bucket_life_cycle_config)
+                    if response.status_code == 200:
+                        log.info("bucket life cycle retrieved")
+                    else:
+                        raise TestExecError("bucket lifecycle config retrieval failed")
+                else:
+                    raise TestExecError("bucket life cycle retrieved")
+                lc_data = json.loads(utils.exec_shell_cmd("radosgw-admin lc list"))
+                log.info(f"lc data is {lc_data}")
+                # uploading data / object
+                if config.test_ops["create_object"] is True:
+                    config.obj_size = 16777216  # 16 * 1024 * 1024
```

Comment:
Cn we make it dynamic using following format in yaml file
objects_size_range:
    min: 0
    max: 0

Reply:
updated as per comments